lib/beaker-pe/pe-client-tools/executable_helper.rb

```diff
@@ -75,7 +75,9 @@ def login_with_puppet_access_on(host, credentialed_dispatcher, opts={})
 
             user = credentialed_dispatcher.credentials.login
             password = credentialed_dispatcher.credentials.password
-            puppet_access_on(host, 'login', {:stdin => "#{user}\n#{password}\n"})
+            args = ['login']
+            args.push "--lifetime #{lifetime}" if lifetime
```

Comment:
If you pass in an array as an argument it acts as a single argument of type array.  If you add the * before the argument, it passes each member of the array as a separate argument.